Library/Homebrew/formula_installer.rb

```diff
@@ -1502,6 +1502,7 @@ def pour
     tab.unused_options = []
     tab.built_as_bottle = true
     tab.poured_from_bottle = true
+    if Hardware::CPU.arm? && Utils::Bottles.on_macos?
```

Comment:
This `if` block seems a bit weirdly placed. Is what's under it indented correctly?